packages/hash/shared/src/entityStorePlugin.ts

```diff
@@ -209,11 +211,6 @@ const updateBlockEntity = (
     throw new Error("draftId provided does not point to a BlockEntity");
   }
 
-  // we shouldn't need to update this since the api is meant to
-  // handle it.
-  // @todo remove the need for this
-  draftBlockEntity.entityVersionCreatedAt = new Date().toISOString();
-
```

Comment:
This was the previous temporary workaround added to prevent the block from resetting to it's previous state after it's data has been swapped